### DISCUSSION TITLE
Ability to change stack order of sprites

### DISCUSSION THREAD

**Discussion by AuracleTech** [ppy/osu#20116] [Ideas] [answered]
![My current layout](https://user-images.githubusercontent.com/40531871/188284846-9996c2ee-bb67-4b09-8cb4-e3bf89a3280c.png)
Trying to add even more combo count because why not, but I can't click the elements without removing my overlay 
![image](https://user-images.githubusercontent.com/40531871/188284850-4e11492a-b793-431c-8af2-e6af3423a39e.png)

Could be nice to add on `Right Click` -> `Send to Background` / `Send to Foreground`

**Comment by nagi-desuuu** [comment, marked as answer]
https://github.com/ppy/osu/discussions/20114

**Reply by AuracleTech** [reply]
I should have made sure it wasn't already open.
